pyoffers/api.py

```diff
@@ -152,7 +152,11 @@ def init_target_object(self, target, data):
         """
         Initializes target object and assign extra objects to target as attributes
         """
-        target_object = self.init_single_object(target, data.pop(target))
+        if isinstance(data, bool):
+            # Some update methods return True OR an instance of a new object.
+            # See: https://developers.tune.com/network/offer-setaffiliateapproval/
+            return data
+        target_object = self.init_single_object(target, data.pop(target, data))
```

Comment:
Why is it required to set `data` as a fallback?

Reply:
In this case response doesn't contain a key named by target instance, but contains a dict with an actual instance.